espnet/nets/pytorch_backend/lm/ngram.py

```diff
@@ -0,0 +1,102 @@
+"""Ngram lm implement."""
+
+import kenlm
+import torch
+
+from espnet.nets.scorer_interface import PartialScorerInterface
+from espnet.nets.scorer_interface import ScorerInterface
+
+from abc import ABC
+
+
+class Ngrambase(ABC):
+    """Ngram base implemented throught ScorerInterface."""
+
+    def __init__(self, ngram_model, token_list):
+        """Initialize Ngrambase.
+
+        Args:
+            ngram_model: ngram model path
+            token_list: token list from dict or model.json
+
+        """
+        self.chardict = [x if x != "<eos>" else "</s>" for x in token_list]
```

Comment:
I have a question.
In espnet, `<eos>` represents `<s>` and </s> both, but you only care about `</s>`. Is this no problem?

Reply:
I have descripted in the program since the https://github.com/qmpzzpmq/espnet/blob/8bf8db9e855ceef181e4fbce4a610724da283c3d/espnet/nets/pytorch_backend/lm/ngram.py#L31

Reply:
`<eos>` represents both `<s>` and `</s>` (=same label is used in espnet) and the hypothesis always start with `<eos>`. 
Am I wrong? I didn't read the detail of the code.

Reply:
please read this one:
https://github.com/kpu/kenlm/blob/87e85e66c99ceff1fab2500a7c60c01da7315eec/python/example.py#L34-L35

Reply:
The examples just mentions the case without `<s>`, but our case, it starts with `<s>`.

Reply:
Yes, I recheck it. I correct it.

Reply:
Thank you, lgtm